lib/src/Base/Stat/LowDiscrepancySequenceImplementation.cxx

```diff
@@ -381,9 +381,9 @@ Unsigned64BitsInteger LowDiscrepancySequenceImplementation::GetNextPrimeNumber(c
 #ifdef OPENTURNS_HAVE_PRIMESIEVE
   primesieve::iterator it;
 #if PRIMESIEVE_VERSION_MAJOR >= 11
-  it.jump_to(n - 1);
+  it.jump_to(n);
```

Comment:
Line 380, the test `if (n == 0) return 2;` should be replaced by `if (n <= 2) return 2;`